python/paddle/fluid/tests/unittests/CMakeLists.txt

```diff
@@ -852,8 +852,8 @@ if(WIN32)
     DEPS python)
 endif()
 
-if(WITH_MKLDNN)
-  add_subdirectory(mkldnn)
+if(WITH_ASCEND_CL)
```

Comment:
why change this ?

Reply:
sry, made a mistake while solving conflicts.
removed both.